fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -74,7 +74,7 @@ def self.description
 
       def self.output
         [
-          ['ORIGINAL_DEFAULT_KEYCHAIN', 'The path to the new system default keychain']
+          ['ORIGINAL_DEFAULT_KEYCHAIN', 'The path to the default keychain']
```

Comment:
Out of scope for this PR, but:
Why is this added to the output here?

Reply:
I think its in scope because its a shared value that wasn't included - https://github.com/fastlane/fastlane/pull/14610/files#diff-0231a345cb1ba3f328959032b555024cR6

Reply:
Nah, I meant the question is out of scope: Why does it make sense to output the original default keychain when the action should create a new one?